src/components/ResourceList/ResourceList.tsx

```diff
@@ -29,15 +29,16 @@ const SMALL_SCREEN_WIDTH = 458;
 const SMALL_SPINNER_HEIGHT = 28;
 const LARGE_SPINNER_HEIGHT = 45;
 
+export type Items = any[];
+
 export interface State {
   selectMode: boolean;
   loadingPosition: number;
-  listNode: HTMLUListElement | null;
 }
 
 export interface Props {
   /** Item data; each item is passed to renderItem */
-  items: any[];
+  items: Items;
```

Comment:
Just curious, what are you fixing by using a type here?

Reply:
We use this type multiple times. We just put `any[]` every time.